dashboard/test/models/block_test.rb

```diff
@@ -15,15 +15,13 @@ class BlockTest < ActiveSupport::TestCase
     assert_equal block.helper_code, seeded_block.helper_code
 
     seeded_block.destroy
-    File.delete "#{base_path}.json"
-    File.delete "#{base_path}.js"
     Dir.rmdir "config/blocks/fakeLevelType"
   end
 
   test 'Block writes to and loads back from file without helper code' do
     block = create :block, helper_code: nil
     json_before = block.block_options
-    block.destroy
+    block.delete
```

Comment:
Why delete these now instead of destroy?

Reply:
I use the block files created for `block` to test seeding later in these tests, so I don't want them to get cleaned up by the `after_destroy` hook.